ext/mysqli/tests/gh8978.phpt

```diff
@@ -5,6 +5,7 @@ mysqli
 --SKIPIF--
 <?php
 require_once 'skipifconnectfailure.inc';
+if (getenv('IS_MARIADB_TEST')) die('skip have not yet configured MariaDB to use SSL');
```

Comment:
Is skipping a good idea? Maybe expect failure instead? What exactly is the issue with this test?

Reply:
Whether SSL is enabled is indicated by the bits in `capability_flags`.

https://github.com/php/php-src/blob/0039446fee48a48896e63d978ba3ac531fd4c859/ext/mysqlnd/mysqlnd_wireprotocol.c#L390

For MySQL, it is turned on without any special settings, but for MariaDB, it is turned off by default.

That is, the if block that should be checked in this test is not reached.

https://github.com/php/php-src/blob/0039446fee48a48896e63d978ba3ac531fd4c859/ext/mysqlnd/mysqlnd_commands.c#L529

It seemed like it would take a while to figure out how to turn this on, so I thought I'd just skip it first and follow up later.

Reply:
Thank you! I will try this method locally.

Reply:
MariaDB11.4 seems to enable SSL by default, which means it behaves the same as MySQL.
https://mariadb.com/kb/en/changes-improvements-in-mariadb-11-4/#ssltls

Therefore, by fixing the MariaDB used for CI to 11.4 or later, there is no need to configure SSL.
However, there is a new issue #13628, so I will wait until it is resolved before proceeding with this PR.